src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,44 @@
+.stopwatch {
+  @include center;
+
+  position: relative;
+  width: $watch-size;
+  height: $watch-size;
+  border: $watch-border;
+  border-radius: 50%;
+
+  &__minute {
+    position: absolute;
+    width: $minute-width;
+    height: $minute-height;
+    background-color: $minute-color;
+    transform: translateY(-50%);
+    animation: $minute-animation-setings;
```

Comment:
The variable `$minute-animation-setings` is not defined in this file. Make sure to define it in your variables file and import it at the top of this file.